src/Microsoft.Tye.Core/ConfigModel/ConfigFactory.cs

```diff
@@ -73,7 +74,7 @@ private static ConfigApplication FromSolution(FileInfo file)
                 {
                     var service = new ConfigService()
                     {
-                        Name = Path.GetFileNameWithoutExtension(projectFile.Name).ToLowerInvariant(),
+                        Name = Normalization(Path.GetFileNameWithoutExtension(projectFile.Name).ToLowerInvariant()),
```

Comment:
nitpick: move `ToLowerInvariant()` into the normalization method so it's not copied all over 👍